tests/unit/libbpf_test.cpp

```diff
@@ -2562,13 +2612,25 @@ TEST_CASE("BPF_MAP_GET_NEXT_KEY etc.", "[libbpf]")
     attr.next_key = (uintptr_t)&next_key;
     REQUIRE(bpf(BPF_MAP_GET_NEXT_KEY, &attr, sizeof(attr)) < 0);
     REQUIRE(errno == ENOENT);
-
+  
     // Delete the entry.
     memset(&attr, 0, sizeof(attr));
     attr.map_fd = map_fd;
     attr.key = (uintptr_t)&key;
     REQUIRE(bpf(BPF_MAP_DELETE_ELEM, &attr, sizeof(attr)) == 0);
 
+    // Look up and delete the entry.
+    memset(&attr, 0, sizeof(attr));
+    value = 0;
+    key = 42;
+    attr.map_fd = map_fd;
+    attr.key = (uintptr_t)&key;
+    attr.value = (uintptr_t)&value;
+
+    // Add the element back to the entry after the previous test entry deletion.
+    bpf(BPF_MAP_UPDATE_ELEM, &attr, sizeof(attr));
+    REQUIRE(bpf(BPF_MAP_LOOKUP_AND_DELETE_ELEM, &attr, sizeof(attr)) == 0);
+
```

Comment:
add another check in here when the element doesn't exist.  I.e., call bpf(BPF_MAP_LOOKUP_AND_DELETE_ELEM, &attr, sizeof(attr)) again and test the return code.